reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -188,7 +188,7 @@ class FreemarkerTemplateProcessor(
         @Suppress("UNUSED") // This function is used in the templates.
         fun filterForCategory(licenses: Collection<ResolvedLicense>, category: String): List<ResolvedLicense> =
             licenses.filter { resolvedLicense ->
-                licenseClassifications[resolvedLicense.license]?.categories?.contains(category) ?: true
+                licenseClassifications[resolvedLicense.license]?.contains(category) ?: true
```

Comment:
One more thing: This fallback to `true` (which was also there in the original code) made me wonder. Shouldn't we fall back to `false`, so that uncategorized licenses are not treated as if they belong to the category?